include/grass/defs/glocale.h

```diff
@@ -1,10 +1,11 @@
 #ifndef GRASS_GLOCALEDEFS_H
 #define GRASS_GLOCALEDEFS_H
 
-#if !defined __GNUC__ || __GNUC__ < 2
-#undef __attribute__
+#if defined(_MSC_VER)
+#ifndef __attribute__
 #define __attribute__(x)
 #endif
+#endif
```

Comment:
I don't feel like this is better, so should we revert to what we have now?

```suggestion
#if !defined __GNUC__ || __GNUC__ < 2
#undef __attribute__
#define __attribute__(x)
#endif
```